week-1/errors/4.js

```diff
@@ -1,2 +1,10 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+// const 12HourClockTime = "20:53";
+// const 24hourClockTime = "08:53";
+
+//corrected example =>
+const _12HourClockTime = "20:53";
```

Comment:
This is technically correct - but variable names starting with _ isn't conventional. Can you think of different variable names?